ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -182,72 +199,126 @@ const POPULATIONS = [
+    fieldTitle: 'This Callset',
+    acDisplay: 'AC',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
+  },
   {
     field: 'gnomad_mito',
     fieldTitle: 'gnomAD mito',
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
+    precision: 3,
+    ...GNOMAD_URL_INFO,
+  },
+  {
+    field: 'gnomad_mito_heteroplasmy',
+    fieldTitle: 'gnomAD mito',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
     precision: 3,
-    urls: { [GENOME_VERSION_38]: 'gnomad.broadinstitute.org' },
-    queryParams: { [GENOME_VERSION_38]: 'dataset=gnomad_r3' },
+    ...GNOMAD_URL_INFO,
   },
   {
     field: 'helix',
     fieldTitle: 'Helix mito',
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
+    precision: 3,
+  },
+  {
+    field: 'helix_heteroplasmy',
+    fieldTitle: 'Helix mito',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
     precision: 3,
   },
 ]
 
+const DETAIL_SECTIONS = [
+  {
+    name: 'Global AFs',
+    hasDetail: pop => pop && pop.filter_af && (pop.filter_af !== pop.af),
+    display: () => [{ subTitle: '', value: 'af' }],
+  },
+  {
+    name: 'Allele Counts',
+    hasDetail: pop => pop && pop.ac,
+    display: () => [{ subTitle: '', value: 'ac' }],
+  },
+]
+
+const MITO_DETAIL_SECTIONS = [
+  {
+    name: HOM_SECTION,
+    hasDetail: pop => pop && pop.ac,
+    display: () => [{ subTitle: '', value: 'ac' }],
+  },
+  {
+    name: HET_SECTION,
+    hasDetail: pop => pop && (pop.ac || pop.max_hl),
+    display: pop => ([
+      pop.ac && { subTitle: '', value: 'ac' },
+      pop.max_hl && { subTitle: ' max observed heteroplasmy', value: 'max_hl' },
+    ].filter(d => d)),
+  },
+]
+
+const DISPLAY_VALUE = {
+  ac: pop => `${pop.ac} out of ${pop.an}`,
+  af: (pop, popConfig) => `${pop.af.toPrecision(popConfig.precision || 2)}`,
+  max_hl: (pop, popConfig) => `${pop.max_hl.toPrecision(popConfig.precision || 2)}`,
+}
+
 const Frequencies = React.memo(({ variant }) => {
   const { populations = {} } = variant
-  const freqContent = <div>{POPULATIONS.map(pop => <FreqSummary key={pop.field} variant={variant} {...pop} />)}</div>
+  const callsetHetPop = populations.callset_heteroplasmy
+  const isMito = callsetHetPop && callsetHetPop.af !== null && callsetHetPop.af !== undefined
+  const popConfigs = isMito ? MITO_POPULATIONS : POPULATIONS
+  const sections = (isMito ? MITO_DETAIL_SECTIONS : DETAIL_SECTIONS).reduce(
+    (acc, section) => ([
+      ...acc,
+      {
+        name: section.name,
+        details: popConfigs.map(popConfig => (section.hasDetail(populations[popConfig.field]) &&
+          (!popConfig.section || popConfig.section === section.name) &&
+          section.display(populations[popConfig.field], popConfig).map(({ subTitle, value }) => (
+            <Popup.Content key={`${section.name}${popConfig.field}${subTitle}`}>
+              {`${popConfig.fieldTitle}${subTitle}: ${DISPLAY_VALUE[value](populations[popConfig.field], popConfig)}`}
```

Comment:
`value` is a confusing name for this property, as it is actually the `valueField` for looking up the `value` to display, not the value itself so I would rename it